src/components/molecules/CategoryEdit/index.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <div class="category-edit">
+    <div v-if="doneMessage" class="category-edit__notice--update">
+      <app-text bg-success>{{ doneMessage }}</app-text>
+    </div>
+    <div class="category-edit__columns">
+      <section class="category-edit-editor">
+        <app-heading :level="1">カテゴリー管理</app-heading>
+        <app-router-link
+          block
+          underline
+          key-color
+          hover-opacity
+          to="/categories"
+          class="category-edit-editor-back"
+        >
+          カテゴリー一覧へ戻る
+        </app-router-link>
+        <div class="category-edit-form">
+          <app-input
+            v-validate="'required'"
+            name="name"
+            type="text"
+            placeholder="カテゴリー名を入力してください。"
+            white-bg
+            data-vv-as="カテゴリー名"
+            :error-messages="errors.collect('name')"
+            :value="categoryName"
+            @update-value="$emit('category-name', $event)"
```

Comment:
[imo]

emit 時のイベント名は、動詞を含めた方が良いかなと思います。
編集中に走るイベントになると思うので、以下のようなイベント名が適切だと思いました。

```suggestion
            @update-value="$emit('editing-category-name', $event)"
```

Reply:
ご指摘の通り修正しました